dask/dataframe/io/csv.py

```diff
@@ -602,6 +602,7 @@ def to_csv(
     mode="wt",
     name_function=None,
     compression=None,
+    compute_kwargs=None,
     compute=True,
     scheduler=None,
```

Comment:
Scheduler is passed in here, but it is also possible to pass it through to `.compute()` via the compute **kwargs.

I think it should belong only as part of the `compute_kwargs`; should I raise a warning if people are passing it in directly here for now, and then in a later release remove `scheduler` as an argument to this function?

Reply:
If its present both on its own, and in the `compute_kwargs`, an error is raised:
```python
TypeError: add() got multiple values for keyword argument 'b'
```
If the same value for `scheduler` is passed in via the `compute_kwargs` and directly, should this just raise a warning for now, or tell the user to only pass it in one way?

Reply:
I think we want to add a deprecation warning about `scheduler` -- there's an example of that here: https://github.com/dask/dask/commit/9748ca2898d592a959df45f8f0abe7fb13db0f2f

--

If `scheduler` is passed in on its own and also via `compute_kwargs` I think that should raise an error, otherwise we're in the business of deciding which of two specified `scheduler`s to use